net.lutris.Lutris.yaml

```diff
@@ -49,6 +49,11 @@ add-extensions:
     enable-if: active-gl-driver
 
 modules:
+- name: gnome-desktop
+  sources:
+  - type: archive
+    url: "https://download.gnome.org/sources/gnome-desktop/3.30/gnome-desktop-3.30.2.tar.xz"
```

Comment:
gnome desktop 3.32?

Reply:
Strangely, 3.32 didn't work for me. You mind trying it out?

Reply:
Here it works